src/components/home/homeHeader/styled.js

```diff
@@ -0,0 +1,273 @@
+import styled from 'styled-components';
+import { media } from '@components/bootstrap-grid';
```

Comment:
'@components...' or 'components...'? SHOULD BE SAME APPROACH TO ALIASES EVERYWHERE